chpl/chpl-service/pom.xml

```diff
@@ -275,10 +270,11 @@
             <artifactId>ff4j-core</artifactId>
             <version>${ff4j.version}</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.ff4j/ff4j-store-redis -->
         <dependency>
             <groupId>org.ff4j</groupId>
-            <artifactId>ff4j-store-ehcache</artifactId>
-            <version>${ff4j.version}</version>
+            <artifactId>ff4j-store-redis</artifactId>
+            <version>2.0.0</version>
```

Comment:
Is it worth moving this version (and the other two in the file) to variables at the top of the file?